data.sql

```diff
@@ -1,5 +1,22 @@
+-- Day 1
+
 insert into animals(name,date_of_birth, weight_kg,neutered,escape_attempts) values ('Agumon','2020-02-03', 10.23,true,0);
 insert into animals(name,date_of_birth, weight_kg,neutered,escape_attempts) values 
   ('Gabumon','2018-11-15', 8,true,2),
   ('Pikachu','2021-01-07',15.04,false,1),
   ('Devimon','2017-05-12', 11,true,5);
+
+-- Day 2
+
+alter table animals add column species varchar(100) ;
```

Comment:
Kindly make sure that everything that creates or changes the table structure should be saved in the schema.sql file. please add `alter table animals add column species varchar(100)`  in the schema.sql as required by the previous reviewer [here](https://github.com/wickathou/vet-clinic-sql-practice/pull/2/files#r1157188850) :+1:

Reply:
I noticed I forgot to push the changes :sweat_smile: